File: src/java1/q5.java
// Find common elements between two arrays.
package java1;

public class q5 {
    public static void main(String[] args) {
        int a1[] = {0,1,2,3,6,7,8};
        int a2[] = {1,6,7,9,2,8,9};
        for (int i = 0; i < a1.length ; i++) {
            for(int j = 0;j < a2.length;j++){
                if(a1[i]==a2[j]){
                    System.out.println(a1[i] + " ");
                }

            }

        }

    }
}

File: Exercise1/q5.kt
package AssesmentOne

//Find common elements between two arrays.

fun main() {
    var array1 = Array(5){0}
    var array2 = Array(4){0}

    println("Enter values")
    var x = 0
    while (x < array1.size) {
        array1[x] = readLine()!!.toInt()
        x++
    }
    println("Enter values")
    var y = 0
    while (y < array2.size) {
        array2[y] = readLine()!!.toInt()
        y++
    }

    println("Common elements are:")
    for (i in 0 until array1.size) {
            for (j in 0 until array2.size)
                if (array1[i] == array2[j])
                    println(array1[i])

        }

}
